lib/reline/config.rb

```diff
@@ -62,10 +62,11 @@ def reset_variables
     @keyseq_timeout = 500
     @test_mode = false
     @autocompletion = false
-    @convert_meta = true if seven_bit_encoding?(Reline::IOGate.encoding)
+    @convert_meta = seven_bit_encoding?(Reline::IOGate.encoding)
```

Comment:
I thought it would be better to use true/false instead of nil and true, so I made the change.